### ISSUE TITLE
Missing Accessibility labels

### ISSUE THREAD

**Issue by ulukaya** [hyperoslo/ImagePicker#388]
1) It starts with focus to "nowhere". May start with camera button or first image in focus.
2) No labels for flash, switch camera or camera action button.

**Comment by ulukaya** [comment]
https://github.com/hyperoslo/ImagePicker/pull/389
